config/locales/en.yml

```diff
@@ -2445,37 +2444,37 @@ en:
     welcome:
       title: Welcome!
       introduction: |
-        Welcome to OpenStreetMap, the free and editable map of the world. Now that you're signed
+        Welcome to OpenHistoricalMap, the free and editable map of world history. Now that you're signed
         up, you're all set to get started mapping. Here's a quick guide with the most important
         things you need to know.
       whats_on_the_map:
         title: What's on the Map
-        on_the_map_html: |
-          OpenStreetMap is a place for mapping things that are both %{real_and_current} -
-          it includes millions of buildings, roads, and other details about places. You can map
-          whatever real-world features are interesting to you.
-        real_and_current: real and current
+        on_the_map_ohm_html: |
+          OpenHistoricalMap tracks changes to natural and human geography throughout the world.
+          We have millions of entries from %{past_and_present}, from empires and glaciers to houses and restaurants.
+          You can map whatever real-world features are interesting to you.
+        past_and_present: both the past and present
```

Comment:
We’ll need to be vigilant about Translatewiki.net’s FuzzyBot attempting to rename `on_the_map_html` to `on_the_map_ohm_html` without replacing `real_and_current` with `past_and_present`.